src/xia2/Modules/MultiCrystal/ScaleAndMerge.py

```diff
@@ -368,6 +379,8 @@ def __init__(self, experiments, reflections, params):
 
         self.decide_space_group()
 
+        self.reindex()
```

Comment:
We should only run this `if params.reference is not None`,

Reply:
Added this :)